docs/bib/use_citations.bib

```diff
@@ -1,3 +1,16 @@
+
+% 2021-04-25
+@article{Angelescu:2021lln,
+    author = "Angelescu, Andrei and Be\v{c}irevi\'c, Damir and Faroughy, Darius A. and Jaffredo, Florentin and Sumensari, Olcyr",
+    title = "{On the single leptoquark solutions to the $B$-physics anomalies}",
+    eprint = "2103.12504",
+    archivePrefix = "arXiv",
+    primaryClass = "hep-ph",
+    reportNumber = "ZU-TH 12/21",
+    month = "3",
+    year = "2021"
```

Comment:
```suggestion
    year = "2021",
    journal = ""
```

Sphinx currently requires a journal. We can probably change that, but this should fix it.